app/presenters/feed_entry_presenter.rb

```diff
@@ -18,7 +18,11 @@ def atom_url
   end
 
   def updated
-    Time.zone.parse(result["public_timestamp"])
+    # Collections now uses London time. Convert it back to it's former zone
+    # of UTC in the domain of atom feeds to ensure backwards compatibility.
+
+    @utc_time = Time.find_zone("UTC")
+    @utc_time.parse(result["public_timestamp"])
```

Comment:
do we need instance variables here? could we get away with

```
Time.find_zone("UTC").parse(result["public_timestamp"])
```

Reply:
yep! Will change